syntax = "proto3";

service ChordService {
    rpc findSuccessor(FindSuccessorRequest) returns (FindSuccessorResponse) {}
}

message FindSuccessorRequest {
    int32 id = 1;
}

message FindSuccessorResponse {
    int32 id = 1;
    string ip = 2;
    bool is_final = 3;
}